.travis.yml

```diff
@@ -7,10 +7,9 @@ cache:
         - $HOME/.composer/cache/files
 
 php:
-    - 7.2
-    - 7.3
```

Comment:
why do you drop these builds? imho we should then also note that in the changelog and adjust composer.json, if it is necessary. if it is free to keep supporting 7.2 and 7.3 i would keep it in for now.

Reply:
> would you have time to rewrite the .travis.yml to a github-action workflow? similar to https://github.com/php-http/client-common/blob/master/.github/workflows/tests.yml

#15